spec/shoryuken/manager_spec.rb

```diff
@@ -8,6 +8,15 @@
     Shoryuken::Manager.new(condvar)
   end
 
+  describe 'Invalid concurrency setting' do
+    it 'raises ArgumentError if concurrency is not positive number' do
+      Shoryuken.options[:concurrency] = -1
+      expect{Shoryuken::Manager.new(nil)}
+      .to raise_error(ArgumentError, 'concurrency value -1 is invalid, the value needs to be a positive number')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.